cli_tools/common/disk/inspect.go

```diff
@@ -31,10 +32,13 @@ type Inspector interface {
 	// Inspect finds partition and boot-related properties for a disk and
 	// returns an InspectionResult. The reference is implementation specific.
 	Inspect(reference string) (InspectionResult, error)
+	GetWorkflow() *daisy.Workflow
```

Comment:
This is a considerable leak of implementation details :)

```suggestion
	Cancel() 
```